README.md

```diff
@@ -10,8 +10,8 @@
 
 ## Install
 
-```batch
-npm i -S metronome-contracts
+```shell
+npm install --save metronome-contracts
```

Comment:
🤔

Reply:
Better to be as descriptive/verbose as possible in the examples. Pro users will `npm i` for sure anyway.